.github/workflows/build.yml

```diff
@@ -8,8 +8,16 @@ on:
 
 jobs:
   ssr:
-    name: Build SSR
+    name: Build SSR (Node ${{ matrix.node }} - ${{ matrix.architecture }})
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - 14
+          - 16
+        architecture:
```

Comment:
I don't know the purpose of this and maybe I'm understanding it wrong, but this doesn't work the way *I think* you want to. I also comment it because right now only x64 is used, but I recall seeing other arches in previous commits.

GitHub Actions runs in the same environment as Azure virtual machines. Specifically, [standard DS2_v2 series](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-general#dsv2-series), so all of it is Intel's

Full details about the environment is detailed here at GH docs: https://docs.github.com/es/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

What you want is something like this: https://github.com/marketplace/actions/run-on-architecture